allspice/apiobject.py

```diff
@@ -795,40 +809,61 @@ def get_file_content(
         else:
             return [Content.parse_response(self.allspice_client, f) for f in self.allspice_client.requests_get(url, data)]
 
-    def get_generated_json(self, content: Union["Content", str], ref: Optional["Ref"] = None) -> dict:
+    def get_generated_json(
+            self,
+            content: Union[Content, str],
+            ref: Optional[Ref] = None
+    ) -> dict:
         """
         Get the json blob for a cad file if it exists, otherwise enqueue
         a new job and return a 503 status.
 
-        Note: This is still experimental and not yet recommended for
-        critical applications.
+        !!!WARNING: Here be dragons!!!
```

Comment:
I don't think the warning message needs to be quite so deadly.

Realistically the JSON is stable. We may change some things like added fields but removed fields isn't as likely.

I would feel comfortable removing "If you are using this, be prep..."

I imagine most customers will be using the scripts provided.

@danielallspice do we have customers who currently rely on the JSON output heavily? Such that a field renaming would break stuff

Reply:
@McRaeAlex Nobody is relying on the JSON output heavily right now